.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 # See https://pre-commit.com for more information
 repos:
   - repo: https://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.10
```

Comment:
updated the versions. v1.1.10 wasnt running on Arm based macs, updating solved it and was able to run locally.

Reply:
Please make things like this a seperate PR.
Also: We don't support ARM in any way, shape or form atm

Reply:
I'll split it into a separate PR, since its not really an ARM specific change and just updating to a newer version i figured it was ok.

Reply:
moved to: https://github.com/truecharts/charts/pull/20460